src/OmniSharp.LanguageServerProtocol/OmniSharp.LanguageServerProtocol.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net46</TargetFramework>
+        <PlatformTarget>AnyCPU</PlatformTarget>
+        <AssemblyName>OmniSharp.LanguageServerProtocol</AssemblyName>
+        <PreserveCompilationContext>true</PreserveCompilationContext>
+        <RuntimeIdentifiers>win7-x64;win7-x86</RuntimeIdentifiers>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <PackageReference Include="OmniSharp.Extensions.LanguageServerProtocol" Version="0.7.0" />
+        <!-- <ProjectReference Include="../../../csharp-language-server-protocol/src/Lsp/Lsp.csproj" /> -->
+        <ProjectReference Include="..\OmniSharp.Host\OmniSharp.Host.csproj" />
+        <ProjectReference Include="..\OmniSharp.Roslyn.CSharp\OmniSharp.Roslyn.CSharp.csproj" />
+        <ProjectReference Include="..\OmniSharp.DotNet\OmniSharp.DotNet.csproj" />
+        <ProjectReference Include="..\OmniSharp.DotNetTest\OmniSharp.DotNetTest.csproj" />
+        <ProjectReference Include="..\OmniSharp.MSBuild\OmniSharp.MSBuild.csproj" />
+        <ProjectReference Include="..\OmniSharp.Script\OmniSharp.Script.csproj" />
```

Comment:
Reference OmniSharp.Cake!

Reply:
? Not ! (Clumsy me on phone)

Reply:
I have a feeling some of those references can go away, because we don't specifically get types for a given project system, we just use the metadata from MEF.

Reply:
however I will validate that in a day or two.

Reply:
I’d think it will not load unless it’s a compile time dependency. I might be wrong though :) https://github.com/OmniSharp/omnisharp-roslyn/blob/master/src/OmniSharp.Host/CompositionHostBuilder.cs#L158

Reply:
It's a dependency via `OmniSharp.Stdio` and we bootstrap `Lsp` through `Stdio`.
https://github.com/OmniSharp/omnisharp-roslyn/blob/lsp/src/OmniSharp.Stdio/Program.cs#L36

We need to make a unified bootstrapper or rename stdio in a future change.